google/cloud/spanner_v1/_opentelemetry_tracing.py

```diff
@@ -72,6 +94,13 @@ def trace_call(name, session, extra_attributes=None):
     if extra_attributes:
         attributes.update(extra_attributes)
 
+    # TODO(@odeke-em) enable after discussion with team and agreement
```

Comment:
Right, but those are the "breaking changes" that you were concerned about which I believe I had mentioned we'd need to document firstly and give customers time to avoid surprises.

Reply:
got it. We can then have `enable_extended_tracing ` to be `True` by default in python spanner.
Customer can override this by setting it to false in their application. We can explicitly mention that this bahaviour is different from other languages to avoid braking changes.

Reply:
Done, roger that, added, documented and also tested @harshachinta!